R/trust_types.R

```diff
@@ -0,0 +1,37 @@
+# This script joins the trust type (from the CQC rating data) onto the open trust data in the geographr package  
+
+# Load libraries 
+library(geographr)
+library(sf)
+library(tidyverse)
+library(readODS)
+
+source("R/utils.R") # for download_file()
+
+# Downloading CQC rating data as has information on what is the primary type of care trust provides 
+# Landing page of data: https://www.cqc.org.uk/about-us/transparency/using-cqc-data#directory
+tf <- download_file("https://www.cqc.org.uk/sites/default/files/04_January_2022_Latest_ratings.ods", "ods")
+
+raw_providers <-
+  read_ods(
+    tf,
+    sheet = "Providers",
+  )
+
+providers <- raw_providers |>
+  distinct(`Provider ID`, `Provider Primary Inspection Category`) 
+
+# Joining onto open trusts geographr data
+open_trusts <-
+  geographr::points_nhs_trusts |>
+  as_tibble() |>
+  filter(status == "open") |>
+  select(
+    trust_code = nhs_trust_code
+  )
+
+open_trust_types <- open_trusts |>
+  left_join(providers, by = c("trust_code" = "Provider ID")) |>
+  rename(primary_category = `Provider Primary Inspection Category`)
```

Comment:
Had noticed that, they also crop up as missing trusts in some of the indicator data (CQC ratings and hospital maintenance backlog). RYT, RT4 and RQF are not found in the PHE data used to map from Trusts to LA (they must be non acute trusts) so their indicator data gets dropped when this step occurs. RW6 has merged into RM3 & R0A (which are both in the dataset), I'll dig to see if there are older versions on this dataset archived to find the category of RW6. RD3 & RDZ have changed to R0D but would expect it to be in the CQC data since is more up to date. Will investigate.

Reply:
Manually added RW6 type in https://github.com/britishredcrosssociety/resilience-index/pull/6/commits/27d5b36d9e2c112e3d2b23a9dda2065e3910c828.